.github/workflows/update-charts-dispatch.yaml

```diff
@@ -32,22 +30,19 @@ jobs:
           version: v3.10.0
 
       - name: Update Helm chart with changes files
-        if: github.event.action != 'trigger-workflow-testkube-agent-main'
+        if: github.event.action != 'trigger-workflow-testkube-agent-main' && github.event.action != 'trigger-workflow-testkube-cloud-ui-main'
         run: |
           # update application version
-           ./scripts/update.sh                              \
-           -c ${{ github.event.client_payload.app }}        \
-           -a ${{ github.event.client_payload.appVersion }} \
-           --verbose
-
-           echo -e "### ${{ github.event.client_payload.app }} Helm chart diff" >> $GITHUB_STEP_SUMMARY
-           echo -e '```' >> $GITHUB_STEP_SUMMARY
-           git --no-pager diff >> $GITHUB_STEP_SUMMARY
-           echo -e '```' >> $GITHUB_STEP_SUMMARY
+          charts=("testkube-cloud-api" "testkube-worker-service" "testkube-logs-service" )
+          for chart in "${charts[@]}"; do
+            ./scripts/update.sh -c "$chart" -a ${{ github.event.client_payload.appVersion }} --verbose
```

Comment:
Just do double check, this takes care of updating the migration image for testkube-cloud-api?